zfsbootmenu/install-helpers.sh

```diff
@@ -151,7 +151,7 @@ create_zbm_profiles() {
 	[ -f /etc/profile ] && source /etc/profile
 	[ -f /lib/zfsbootmenu-completions.sh ] && source /lib/zfsbootmenu-completions.sh
 
-	export PS1="\[\033[0;33m\]zfsbootmenu\[\033[0m\] \w > "
+	export PS1='\$( /libexec/recovery-error-printer )\[\033[0;33m\]zfsbootmenu\[\033[0m\] \w > '
```

Comment:
```suggestion
	export PS1='\[\033[0;33m\]zfsbootmenu\[\033[0m\] \w > '
	export PROMPT_COMMAND="/libexec/recovery-error-printer"
```

Reply:
You can tell I don't actually use Bash as an interactive shell. I didn't know `PROMPT_COMMAND` existed.

Reply:
That's fair. We all know bash is and forever will be a programming language first.